### PR TITLE
Add First Subsystem, Command, and Binding

### PR DESCRIPTION
**Pull request by EthanMuchnik**
Proofread pls

### CODE REVIEW

File: frc/1. Basics/5a. First Subsystem (Roller Intake).md [hunk 1]
```diff
@@ -1,7 +1,200 @@
...
+public abstract class SnailSubsystem extends SubsystemBase {
+
+    public abstract void update();
+    public abstract void outputValues();
+    public abstract void setUpConstantTuning();
+    public abstract void getConstantTuning();
+}
+```
+
+**Now for the line by line review:**
```
Comment (rcya1) [inline frc/1. Basics/5a. First Subsystem (Roller Intake).md]: For this part the line by line is fine but when you actually get into programming a subsystem which people should be understanding more deeply so they can do it for their own subsystems, imo this style is bad

**Review by rcya1 — CHANGES_REQUESTED** [review]
Overall, I dislike the style where you just throw a ton of code at people and then explain what it does. I generally find this really uneffective and hard to follow. I prefer to instead do an actual process of what someone programming a subsystem might choose to do. For instance, going through
1. Putting in and setting up the appropriate motor controllers
2. Defining the correct states
3. Creating the states and adding them to the enum and the update() function
4. Creating the functions to go into those states
Look at my examples from stuff such as outputting values to shuffleboard or the PID one for examples

**Comment by rcya1** [comment]
FYI to anyone looking, do NOT merge this yet (I'm looking at you Matt) because we still need to work on the other sections (this should be a draft PR but oh well).

**Comment by matth101** [comment]
Nice job overall.
